code/src/pages/cart/CartPage.tsx

```diff
@@ -0,0 +1,127 @@
+import { useState } from "react";
+
+import AppBox from "@/components/app-box/AppBox";
+import AppButton from "@/components/app-button/AppButton";
+import AppTypography from "@/components/app-typography/AppTypography";
+
+import { items } from "@/pages/cart/CartPage.constants";
+import CartItem from "@/pages/cart/components/CartItem";
+import formatPrice from "@/utils/format-price/formatPrice";
+
+import "@/pages/cart/CartPage.scss";
+
+type CartItemsType = {
+  image: string;
+  name: string;
+  productPrice: number;
+  quantity: number;
+  calculatedPrice: number;
+};
+
+const CartPage = () => {
+  const [cartItems, setCartItems] = useState<CartItemsType[]>(items);
```

Comment:
It should be inside the redux store. Create the slice for storing the data inside

Reply:
> It should be inside the redux store. Create the slice for storing the data inside

Please check the description to the PR before and then make review